morf-core/src/main/java/org/alfasoftware/morf/jdbc/SqlScriptExecutorProvider.java

```diff
@@ -31,19 +32,23 @@ public class SqlScriptExecutorProvider implements Provider<SqlScriptExecutor> {
 
   private final DataSource dataSource;
   private final Provider<SqlDialect> sqlDialect;
+  private ConnectionResources connectionResources;

```

Comment:
This can be final

Reply:
Discussed with Martyn - leaving this as is, because we’ve left pre-existing public constructors that don’t accept ConnectionResources. Setting this to final will therefore cause compiliation errors.